### PR TITLE
Identity depends on network attachment point

### PR DESCRIPTION
**Pull request by klarose**
Fixes Issue #12

### CODE REVIEW

File: draft-ietf-capport-architecture.xml [hunk 1]
```diff
@@ -559,6 +559,10 @@ o . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . o
                  <t>Be Visible to the API</t>
                  <t>Be Visible to the Enforcement Device</t>
              </list>
+
+             An identifier only needs to apply to the current point of network attachment. If the
```
Comment (martinthomson) [inline draft-ietf-capport-architecture.xml]:
```suggestion
             An identifier might only apply to the current point of network attachment. If the
```

File: draft-ietf-capport-architecture.xml [hunk 2]
```diff
@@ -559,6 +559,10 @@ o . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . o
...
+             device moves elsewhere in the network, thus changing its identifier, its identity from
```
Comment (martinthomson) [inline draft-ietf-capport-architecture.xml]:
```suggestion
             device moves to a different network location its identity could change.```

File: draft-ietf-capport-architecture.xml [hunk 3]
```diff
@@ -559,6 +559,10 @@ o . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . o
...
+             the perspective of other components MAY change.
```
Comment (martinthomson) [inline draft-ietf-capport-architecture.xml]:
```suggestion
```

**Comment by klarose** [comment]
@martinthomson Thanks for the quick review.

**Pull request merged**